spec/requests/waste_exemptions_engine/front_office_edit_declaration_forms_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteExemptionsEngine
+  RSpec.describe FrontOfficeEditDeclarationFormsController do
```

Comment:
Should this have a string describe block like the other request specs?

Reply:
Yes, the request spec shouldn't assume anything about implementing classes - fixed